README.md

```diff
@@ -98,3 +98,24 @@ The code for adding a source could look like this:
 WinGetSourceManager sourceManager = new WinGetSourceManager();
 sourceManager.AddSource("msstore", "https://storeedgefd.dsx.mp.microsoft.com/v9.0", "Microsoft.Rest");
 ```
+
+### Find Latest Versions of a Package:
+
+Using the ***WinGetPackageManager*** class you can use the `SearchPackage` capability to get the latest version of a package and then retrieve the version number from the `AvailableVersion` property.
+
+You would then be able to compare this to the latest version of your executing assembly and determine if you need to notify users of an available upgrade.
+
+```csharp
+Version latestPackageVersion = null;
+WinGetPackageManager packageManager = new WinGetPackageManager();
+string packageId = "nkdAgility.AzureDevOpsMigrationTools";
+var package = packageManager.SearchPackage(PackageId).SingleOrDefault();
```

Comment:
Searching the winget repository is correct. That's where any new versions would be stored. 

I don't see the value in checking locally.

Unless you are suggesting that Winget auto updates the available online versions when new ones become available automatically.

Reply:
I’m sorry I might have misunderstood your intentions with the way you wanted to implement it.  

Let's say you want to get the version info for 7Zip and the installed version is 23.0.
If you use the search function you will only get one version back, which will be the newest, lets say 24.0. The package version and the available version would be the same in the returned object.
When getting the installed package winget will output both versions. You will have the package version (23.0) and the available version (24.0) in the returned object.
The available version should always be up to date even if you just list the installed packages.

The way you implemented it should be fine for your tool, because you check the available version against the assembly version. But I think that for the doc’s of the library it makes more sense to check the package version against the available version, because this will be easier for new users to understand and implement when they work with multiple packages.

Reply:
Does Winget update the latest version data every time it gets the local version?

If not, on what cadence does it update that cache? Would it not only be if I call `winget update` and it refreshes the cache?

In which case I would need to call update, and then the local call for the value to be updated.

I believe the only way to guarantee that the latest version is represented and up to date is a server call.

Reply:
So I just checked https://learn.microsoft.com/en-us/windows/package-manager/winget/settings and it looks like the cache is auto-updated every 5 minutes. So I except that you are right, the local data is good enough and it would be better to call that.

I'll also likely update my tool as well as 5m is 🤷‍♂️ defo fine.